packages/notebook/src/celllist.ts

```diff
@@ -204,7 +209,7 @@ class CellList implements IObservableUndoableVector<ICellModel> {
    */
   set(index: number, cell: ICellModel): void {
     // Generate a new uuid for the cell.
-    let id = uuid();
+    let id = (cell as any).modelDB.basePath;
```

Comment:
Yes, this pattern was more of a temporary hack until we could have a discussion about it. Do you see any negative consequences adding a cell id to the model?